interoptest/src/nodejsservice/server.js

```diff
@@ -64,10 +64,17 @@ function enableHttpPlugin (tracer) {
 function enableJaegerTraceExporter (tracer) {
   // 1. Define service name and jaeger options
   const service = 'nodejsservice';
+  let host = 'localhost';
+  let port = 6832;
+  const jaegerAddr = process.env.JAEGER_SERVICE_ADDR;
```

Comment:
can you add following two lines in kubernetes-manifests/nodejsservice.yaml file.
         env:
         - name: PORT
           value: "10301"
+        - name: JAEGER_SERVICE_ADDR
+          value: "traceui:14268"
         resources:

Reply:
Done.